vtiger_connector_crm/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
 {
-    'name': 'CRM VTiger Integration',
-    'version': '10.0.1.0.0',
-    'summary': 'CRM VTiger Integration',
-    'description': """
-        CRM VTiger Integration""",
+    'name': 'VTiger Base Connector Lead',
```

Comment:
module name and category badhe correct karo

Reply:
ok

Reply:
**Done**